src/resources/parser/glb-container-resource.js

```diff
@@ -190,6 +191,22 @@ class GlbContainerResource {
         return GlbContainerResource.createSceneHierarchy(sceneClones, 'Entity');
     }
 
+    // get material variants
+    getMaterialVariants() {
```

Comment:
do we need to expose this? how do you see it being used?

Reply:
On the user side, we need to provide some API to query the variants that exist in the GLB.